redisinsight/api/src/__mocks__/cloud-subscription.ts

```diff
@@ -67,3 +71,111 @@ export const mockCloudSubscriptionFixed = Object.assign(new CloudSubscription(),
   type: CloudSubscriptionType.Fixed,
   price: mockCloudCapiSubscriptionFixed.price,
 });
+
+export const mockCloudCapiCloudRegion1: ICloudCapiSubscriptionCloudRegion = {
```

Comment:
If you receive data from sm api we must use *Api* not *Capi* in model names. So it will be clear where data came from

Reply:
renamed